src/engraving/style/defaultstyle.cpp

```diff
@@ -47,15 +48,37 @@ DefaultStyle* DefaultStyle::instance()
     return &s;
 }
 
+static bool defaultPageSizeIsLetter()
+{
+    // try PAPERSIZE environment variable
+    char* papersize = getenv("PAPERSIZE");
+    if (papersize) {
+        std::string letter = "letter";
+        return papersize == letter;
+    }
+    // try locale
+    switch (QLocale::system().country()) {
+    case QLocale::UnitedStates:
+    case QLocale::Canada:
+    case QLocale::Mexico:
+    case QLocale::Chile:
+    case QLocale::Colombia:
+    case QLocale::CostaRica:
+    case QLocale::Panama:
+    case QLocale::Guatemala:
+    case QLocale::DominicanRepublic:
+    case QLocale::Philippines:
```

Comment:
Isn't it Nicaragua, El Salvador, Belize, Puerto Rico and Venzuela too?
As far as I can tell the Dominical Republic ises A4

![image](https://user-images.githubusercontent.com/1786669/201535064-2d0f69de-917e-4e76-a543-d18dafbbcee6.png)

Reply:
My list is based on the Wikipedia article on letter size - https://en.m.wikipedia.org/wiki/Letter_(paper_size). But I’m happy to update based on more authoritative documentation.

Reply:
Mine on the Image from https://en.wikipedia.org/wiki/Paper_size